pkg/commands/compute/build_test.go

```diff
@@ -444,6 +444,33 @@ func TestBuildJavaScript(t *testing.T) {
 		t.Skip("Set TEST_COMPUTE_BUILD_JAVASCRIPT or TEST_COMPUTE_BUILD to run this test")
 	}
 
+	// We're going to chdir to a build environment,
```

Comment:
The changes to this file can be summarised as follows...

Define a single temp environment rather than having a new environment for each test scenario.